### ISSUE TITLE
Imported Group e-mails with "Import Emails Automatically = No" are untraceable

### ISSUE THREAD

**Issue by ebogaard** [salesagility/SuiteCRM#6534]
#### Issue
When selecting "Import Emails Automatically" in a Group Email Account, Emails are imported automatically and are then shown in the History-subpanel of Leads/Contacts/etc.

But as not all e-mails are sent to known Contacts/Leads/etc, those mails are not shown in a subpanel.
When you have disabled the option to "Leave messages on server", e-mails are still imported and deleted from the mailbox, but there is no way to view e-mails that aren't relatable to a Contacts/Leads/etc.

#### Expected Behavior
In 7.8, with the old Emails-interface, you could always view all e-mails that were imported from a mail box. If there were some unknown e-mail addresses used, you could inspect those add new Contacts/Leads and/or new e-mail address to existing Contacts/Leads to fix those unrelated e-mails.

#### Actual Behavior
In 7.10, there is no such option when you have disabled the option to "Leave messages on server". As far a I can see, there is no way to view e-mails that are imported, but not linked to another entity.

#### Steps to Reproduce

1. Create a group Imbound Email. Set "Import Emails Automatically" to "No" and "Import Emails Automatically" to "Yes".
2. Send e-mail to this mailbox with an unknown (to the CRM) e-mail address.
3. Check in the DB of SuiteCRM that this e-mail has been imported, in the mailbox that the e-mail has been deleted.
4. See that there is no way in the SuiteCRM-interface to get to this e-mail.

#### Your Environment

* SuiteCRM Version used: 7.10.10
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)):
* Environment name and version (e.g. MySQL, PHP 7):
* Operating System and version (e.g Ubuntu 16.04):

**Comment by Mausino** [comment]
@ebogaard yes, we're fighting with same problem. We're calling the email as ghost emails 👎 Here is great problem that you're not able in filter check "only imported"
